nginx.conf

```diff
@@ -0,0 +1,43 @@
+
+error_log  logs/error.log  debug;
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+
+    sendfile        on;
+    #tcp_nopush     on;
+
+    keepalive_timeout  65;
+    #gzip  on;
+
+    # DataDome configuration
+    lua_ssl_verify_depth 3;
+    lua_ssl_trusted_certificate /etc/ssl/certs/ca-certificates.crt;
+
+    resolver 8.8.8.8;
+    init_by_lua_block {
+        require("datadome_openresty").setDatadomeConf( { datadome_server_side_key = "datadome_server_side_key", } )
```

Comment:
I am being picky but the fact that the two strings were different ( `datadome_server_side_key` and `server_side_key`) before made it easier to understand and search for what should be changed.
In this case we could change the instructions in the README.md to

> Open the file `nginx.conf` and replace `server-side-key` with the value of the DataDome server-side key that you can find in your ...

Reply:
Too late haha